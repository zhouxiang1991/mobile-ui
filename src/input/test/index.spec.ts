import { mount } from '@vue/test-utils'
import { Input } from '..'

test('should render input', () => {
  const wrapper = mount(Input, { })

  expect(wrapper.html()).toMatchSnapshot()
})
