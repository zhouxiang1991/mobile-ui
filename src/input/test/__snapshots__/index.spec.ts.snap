// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render input 1`] = `
"<input class=\\"zx-input dd1\\">
<div class=\\"zx-common\\"> 公用组件 </div>"
`;
